spec/components/guardian/user_guardian_spec.rb

```diff
@@ -492,4 +492,15 @@
       end
     end
   end
+
+  describe "#can_upload_external?" do
+    it "is true by default" do
+      expect(Guardian.new(user).can_upload_external?).to eq(true)
+    end
+
+    it "is false if the user has been banned from external uploads for a time period" do
+      user.ban_from_external_uploads!
```

Comment:
Just want to be sure, are we cleaning up the state here after the test?

Reply:
I assumed we would clear redis after each spec, but apparently that is not the case! I find that very odd but maybe it's done for a good reason?

Reply:
Anyway I will make sure to clean up.

Reply:
I think the problem is that there isn't a way for us to flush Redis just for the test environment. There is a discussion about this on dev somewhere which I can no longer find lol